app/views/tag_set_nominations/_review.html.erb

```diff
@@ -12,26 +12,26 @@
   <ul class="notes">
     <li>You can approve (<span class="yes">+</span>) or reject (<span class="no">x</span>) nominations, or leave them for later. Once a nomination
   has been either approved or rejected, it won't appear on this page again.</li>
-  
-    <li>If you reject a fandom nomination, we will automatically reject all character and relationship nominations submitted 
+
+    <li>If you reject a fandom nomination, we will automatically reject all character and relationship nominations submitted
     for that fandom. We will <em>not</em> automatically approve character nominations.</li>
-    
+
     <li>If you approve non-canonical fandoms and characters, you will have to set up associations for them.</li>
 
     <li>Old (several months) nominations are sometimes cleared from the database. Please review
-    nominations regularly or turn them off.</li>  
+    nominations regularly or turn them off.</li>
   </ul>
 
   <%= form_for :tag_set_nominations, :url => update_multiple_tag_set_nominations_path(@tag_set), :html => { :method => :put, :class => "tagset review" } do |form| %>
-    
+
     <%= error_messages_formatted @errors %>
 
     <% @tagnames_seen = {} %>
 
     <% %w(cast fandom character relationship freeform).each do |tag_type| %>
       <% if @nominations[tag_type] && @nominations[tag_type].count > 0 %>
         <fieldset class="<%= tag_type %> listbox group">
-          <% heading = tag_type == "cast" ? ts("Already Approved Fandoms") : ts("#{tag_type.classify.pluralize} (%{count} left to review)", :count => @tag_set.send("#{tag_type}_nominations").unreviewed.count) %>
+          <% heading = tag_type == "cast" ? ts("Already Approved Fandoms") : ts("#{tag_type_label_name(tag_type).pluralize} (%{count} left to review)", :count => @tag_set.send("#{tag_type}_nominations").unreviewed.count) %>
```

Comment:
Can we have this in the t way for consistency ?

Reply:
Not t, but it wouldn't hurt to switch to % instead of # for consistency:
```suggestion
<% heading = tag_type == "cast" ? ts("Already Approved Fandoms") : ts("%{tag_type} (%{count} left to review)", tag_type: tag_type_label_name(tag_type).pluralize, count: @tag_set.send("#{tag_type}_nominations").unreviewed.count) %>
```